Update application dates for 2025 cycle (#4199)
DFE-Digital/get-into-teaching-app: app/views/content/how-to-apply-for-teacher-training/when-to-apply-for-teacher-training.md

## Deadlines for applications and training provider decisions 

### $recruitmentcycle_deadlinedateonly$

This is the last day you’ll be able to apply for a postgraduate teacher training courses starting in $recruitmentcycle_coursestart$. You must submit your application before $recruitmentcycle_deadlinetimeonly$.

### $recruitmentcycle_providerresponsedeadline$

This is the last day for training providers to make a decision on all applications for courses starting in $recruitmentcycle_coursestart$. 

> If a decision has not been made by the end of September, your applications will automatically be rejected so that you can start applying for next year’s courses in October. 

Reviewer: ```suggestion
If a decision has not been made by this date, your applications will automatically be rejected so that you can start applying for next year’s courses when applications open again. 
```

I wonder if it is clearer to tweak the wording here (and improve mine!) - but would need checking with the Apply team

Author: I've tweaked this line to add in variable values for this so we can be super specific. I think using terms like 'this date' could be a bit confusing in a page where we have lots of different dates flying around. Let me know what you think and if we need to confirm the change with the apply team.
